instrumentation/action_view/lib/opentelemetry/instrumentation/action_view/span_subscriber.rb

```diff
@@ -6,10 +6,27 @@
 
 module OpenTelemetry
   module Instrumentation
+    # rubocop:disable Style/Documentation
     module ActionView
       # The SpanSubscriber is a special ActiveSupport::Notification subscription
       # handler which turns notifications into generic spans, taking care to handle
       # context appropriately.
+
+      # A very hacky way to make sure that OpenTelemetry::Instrumentation::ActionView::SpanSubscriber
+      # gets invoked first
+      def self.subscribe(pattern = nil, callable = nil)
+        ActiveSupport::Notifications.subscribe(pattern, callable)
```

Comment:
1. vanilla subscribe
2. reach inside and push the span_subsciber to the front

Reply:
Is it safe to assume that between subscribing and reaching in to pop the subscriber out that it will definitely be the last item appended to the array?

Reply:
yeah I skimmed the sources

Reply:
If we really wanted to be defensive, we could check to see that the thing we `pop` off is what we expect. But I don't know that we need to.